src/appleseed-max-impl/appleseedinteractive/interactivesession.cpp

```diff
@@ -37,22 +37,19 @@
 #include "renderer/api/project.h"
 #include "renderer/api/rendering.h"
 
-// 3ds Max headers.
-#include <interactiverender.h>
-
-// Forward declarations.
-class Bitmap;
+namespace asf = foundation;
+namespace asr = renderer;
```

Comment:
I could not find any use of asf in this file. But probably should still stay for future use.

Reply:
Oh ok, well we can keep it or remove it, either way it's not a big deal. The main point is to not create namespace aliases in header files.